sdk/identity/identity/Troubleshooting.md

```diff
@@ -58,43 +95,30 @@ azure:identity:info EnvironmentCredential => Invoking ClientSecretCredential wit
 
-To utilize the desired authentication type via `EnvironmentCredential`, ensure the following environment variables are configured properly and that the app can read them.
+## Troubleshoot default Azure credential authentication issues
 
-##### ClientSecret credential
+### Credential unavailable
 
-| Variable Name         | Value                            |
-| --------------------- | -------------------------------- |
-| `AZURE_CLIENT_ID`     | ID of an Azure AD app.           |
-| `AZURE_TENANT_ID`     | ID of the app's Azure AD tenant. |
-| `AZURE_CLIENT_SECRET` | One of the app's client secrets. |
+The `DefaultAzureCredential` attempts to retrieve an access token by sequentially invoking a chain of credentials. In this scenario, the `CredentialUnavailableError` signifies that all credentials in the chain failed to retrieve the token in the current environment setup/configuration. You need to follow the configuration instructions for the respective credential you're looking to use via the `DefaultAzureCredential` chain, so that the credential can work in your environment.
 
-##### ClientCertificate credential
+| Error |Description| Mitigation |
+|---|---|---|
+|`CredentialUnavailableError` thrown with message. "DefaultAzureCredential failed to retrieve a token from the included credentials."|All credentials in the `DefaultAzureCredential` chain failed to retrieve a token, each throwing a `CredentialUnavailableError` themselves|<ul><li>[Enable logging](#enabling-and-configuring-logging) to verify the credentials being tried, and get further diagnostic information.</li><li>Consult the troubleshooting guide for underlying credential types for more information.</li><ul><li>[EnvironmentCredential](#troubleshoot-environment-credential-authentication-issues)</li><li>[ManagedIdentityCredential](#troubleshoot-managed-identity-authentication-issues)</li><li>[VisualStudioCodeCredential](#troubleshoot-visual-studio-code-authentication-issues)</li><li>[VisualStudioCredential](#troubleshoot-visualstudio-credential-authentication-issues)</li><li>[AzureCLICredential](#troubleshoot-azure-cli-authentication-issues)</li><li>[AzurePowershellCredential](#troubleshoot-azure-powershell-authentication-issues)</li></ul>|
```

Comment:
Here it says `thrown with message. “Defau[…]` That dot before the quote is weird. Also, can we use the code quote where the message is? So that the message can look as if it was copied as is into the document?

> `CredentialUnavailableError` thrown with message: `DefaultAzureCredential failed to retrieve a token from the included credentials.`

What do you think?